docs/ai/conceptual/vector-databases-support-semantic-memory.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Using Vector Databases to Extend LLM Capabilities"
+description: "Learn how vector databases extend LLM capabilities by storing and processing embeddings in .NET."
+author: catbutler
+ms.topic: concept-article #Don't change.
+ms.date: 04/12/2024
+
+#customer intent: As a .NET developer, I want to learn how vector databases store and process embeddings in .NET so I can make more data available to LLMs in my apps.
+
+---
+
+# Understanding vector databases in .NET
+
+This article explains how vector databases help you use [embeddings](embeddings.md) to extend the data available to LLMs in .NET.
+
+You can use a vector database to store embeddings that you generate with AI embedding models. Embeddings have dimensions that correspond to the learned features or attributes of the embedding model you use. An embedding contains a value for each dimension, providing a semantic and mathematical representation of the source text.
+
+Vector databases can store embeddings for text, images, and other data types. You can then perform vector analysis on the embeddings to find semantic similarities in the source data, unlocking numerous AI use-cases.
+
+## Available vector database solutions
+
+You can use the following resources as vector database solutions in .NET:
+
+| Resource | SK support | AOAI support |
+|:-|:-|:-|
+| [Azure AI Search](/azure/search/vector-search-overview) | y | y |
```

Comment:
I don't think y/n will make sense in localized content. I suggest using ✔️ and ❌ instead.

Reply:
Thank you! Wasn't sure how to use them, got it now.